MonkeyWrench.Web.WebService/Scheduler/Scheduler.cs

```diff
@@ -120,7 +120,7 @@ public static bool ExecuteScheduler (bool forcefullupdate)
 				reports = GetReports (forcefullupdate);
 
 				using (DB db = new DB (true)) {
-					lanes = db.GetAllLanes ();
+					lanes = db.GetAllLanes ().FindAll(lane => lane.enabled);

```

Comment:
I don't know if we care to see that a lane is disabled in the logs -- it often times hides more useful information.

It is redundant; if you'd prefer not to keep the logging, I can take out the filter in the for-loop.